source/blog/2017-06-22-introducing-buildah.html.md

```diff
@@ -0,0 +1,258 @@
+757b5b595f0a 54e012885dcf first-new-image first-new-image-working-container
+```
+
+Let's mount the container on a local mount point:
+
+```
+# mountpoint2=$(buildah mount $container2)
+```
+
+Now check to see if our `moveit.txt` file is in the new container that we created from the image.
+
+```
+# ls $mountpoint2/tmp  # Let's see if our file is there
+anaconda-post.log  ks-script-64xpp1a9  ks-script-sw0bp7j4  moveit.txt
+# cat $mountpoint2/tmp/moveit.txt
+Move this file to the Buildah container's /tmp directory.
+```
+
+With just a few very scriptable steps we were able to create an image from the container that we had built and modified to fit our needs.  With this image in hand, we can then generate as many new containers as we need in the future based on that image.
+
+## Using Dockerfiles
+
+I'm sure folks with a lot more container time under their belts than I will have many Dockerfile specification files lying around.  These too can be used by Buildah to create a new image.  For instance let's look at this simple Dockerfile that runs a little “Hello World” python script.
+
+```
+FROM python
+ADD HelloFromContainer.py /
+CMD ["python","./HelloFromContainer.py"]
+```
+
+The HelloFromContainer.py file simply contains:
+
+```
+#!/usr/bin/env python3
+#
+
+import sys
+
+def main(argv):
+    for i in range(0,10):
+        print ("Hello World from Container Land! Message # [%d]" % i)
+
+if __name__ == "__main__":
+    main(sys.argv[1:])
+```
+
+Let's put it all together and run the container.  First we'll create an image based on our DockerFile.
+
+```
+# ls HelloFromContainer.py
+HelloFromContainer.py
+# ls Dockerfile
+Dockerfile
+# buildah bud -t hellofromcontainer .
+STEP 1: FROM python
+Getting image source signatures
+Copying blob sha256:ef0380f84d05d3cdc5a5f66023 ...
+ 49.55 MiB / 50.13 MiB [==========================]
+...
+Copying config sha256:74145628c3310c1a8634aa04 ...
+ 0 B / 6.91 KiB [---------------------------------]
+Writing manifest to image destination
+Storing signatures
+STEP 2: ADD HelloFromContainer.py /
+STEP 3: CMD ["python","./HelloFromContainer.py"]
+STEP 4: COMMIT containers-storage:[overlay@/var/lib/containers/storage]docker.io/library/hellofromcontainer:latest
+ 6.91 KiB / 6.91 KiB [============================]
+```
+
+For those wondering at home, this `buildah bud` command completed in 2 minutes and 3 seconds.  The first two minutes of that time was spent in `STEP 1` which pulls down the bits to run Python.  Now let's check that the image got created.
+
+```
+# buildah images
+IMAGE ID        IMAGE NAME
+dc3a60619440 docker.io/library/fedora:latest
+54e012885dcf docker.io/library/first-new-image:latest
+35de96ee8fb5 docker.io/library/python:latest
+f19ac91f7e60 docker.io/library/hellofromcontainer:latest
+```
+
+And let's build the container from that image:
+
+```
+# buildah from hellofromcontainer
+hellofromcontainer-working-container
+# buildah containers
```

Comment:
No Hablo Sweeney-ese, No Problemo!  ;^)

Please change to: 
CONTAINER ID IMAGE ID     IMAGE NAME         CONTAINER NAME
757b5b595f0a 54e012885dcf first-new-image    first-new-image-working-container
8cf6034f22e3 f19ac91f7e60 hellofromcontainer hellofromcontainer-working-container

and hopefully the md compiler will make it all line up nicely under each column.